frontend/src/App.tsx

```diff
@@ -40,51 +40,41 @@ Amplify.configure({
     : undefined
 });
 
-const theme = createMuiTheme({
-  palette: {
-    primary: {
-      main: '#122D5D'
-    }
-  }
-});
-
 const App: React.FC = () => (
   <Router>
-    <ThemeProvider theme={theme}>
+    <CFThemeProvider>
       <AuthContextProvider>
         <Layout>
           <Switch>
-            <AuthRoute
+            <RouteGuard
               exact
               path="/"
-              authComponent={Dashboard}
-              unauthComponent={AuthLogin}
+              component={Dashboard}
+              unauth={AuthLogin}
             />
 
+            <Route
+              exact
+              path="/login-gov-callback"
```

Comment:
I think the login gov callback was handled by a different route earlier (I think it was `/`, but don't remember). What logic makes sure that once someone logs in through login.gov, they are redirected to the /login-gov-callback route?

Reply:
@cablej was able to update the authorized callbacks list within login.gov, I turned off Cognito and was able to login in development.

Reply:
Got it, thanks for updating dev.env.example -- makes it a lot clearer now.

Reply:
Np, good catch forgot to update env there as well